src/esp/physics/ArticulatedObject.h

```diff
@@ -437,6 +443,15 @@ class ArticulatedObject : public esp::physics::PhysicsObjectBase {
     return ids;
   }
 
+  /**
+   * @brief Find the link ID for the given link name
+   */
+  int getLinkIdFromName(const std::string& _name) const {
+    auto linkIdIter = linkNamesToIDs_.find(_name);
+    CORRADE_INTERNAL_ASSERT(linkIdIter != linkNamesToIDs_.end());
```

Comment:
Should we be using ESP_CHECK instead for assertions or is there a reason not to here?

Reply:
I'm just using the same mechanism that is set for the getLink code above - it was coded with the expectation that the query would be correct.

That being said, it does seem rather unforgiving, but I am guessing this is more along the lines of the "fail hard to let folks know" paradigm.

Reply:
On the other hand, getLinkName fails with an ESP_CHECK if the ID is not present. So I'm not sure which to use.